specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/preview/2023-05-01-preview/migrate.json

```diff
@@ -12185,6 +12309,122 @@
+          "items": {
+            "$ref": "#/definitions/EstimatedExternalStorage"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        },
+        "estimatedNetwork": {
+          "type": "array",
+          "description": "Estimated Network",
+          "items": {
+            "$ref": "#/definitions/EstimatedNetwork"
+          },
+          "readOnly": true,
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "AvsEstimatedExternalStorage": {
+      "type": "object",
+      "description": "Details on the Estimated External Storage for AVS Assessment.",
+      "properties": {
+        "monthlyPrice": {
+          "type": "number",
+          "format": "float",
+          "description": "Total monthly cost for type of storage."
+        },
+        "storageType": {
+          "$ref": "#/definitions/ExternalStorageType",
+          "description": "Recommended External Storage."
+        },
+        "totalStorageInGB": {
+          "type": "number",
+          "format": "float",
+          "description": "Predicted total Storage used in GB."
+        },
+        "storageUtilization": {
+          "type": "number",
+          "format": "float",
+          "description": "Predicted storage utilization."
+        }
+      }
+    },
+    "AvsEstimatedNetwork": {
+      "type": "object",
+      "description": "Details on the Estimated Network Costs for AVS Assessment.",
+      "properties": {
+        "monthlyPrice": {
+          "type": "number",
+          "format": "float",
+          "description": "monthly cost for network type."
+        },
+        "networkType": {
+          "$ref": "#/definitions/NetworkSkuType",
+          "description": "Recommended Network Sku."
+        }
+      }
+    },
+    "AvsEstimatedNode": {
+      "type": "object",
+      "description": "Details on the Estimated nodes for AVS Assessment.",
+      "properties": {
+        "monthlyPrice": {
+          "type": "number",
+          "format": "float",
+          "description": "Total monthly cost for type and number of nodes."
+        },
+        "nodeType": {
+          "$ref": "#/definitions/AzureAvsNodeType",
+          "description": "Recommended SKU."
+        },
+        "nodeNumber": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of nodes that will be needed."
+        },
+        "cpuUtilization": {
+          "type": "number",
+          "format": "float",
+          "description": "Predicted CPU utilization."
+        },
+        "ramUtilization": {
+          "type": "number",
+          "format": "float",
+          "description": "Predicted RAM utilization."
+        },
+        "storageUtilization": {
+          "type": "number",
+          "format": "float",
+          "description": "Predicted storage utilization."
+        },
+        "totalCpu": {
+          "type": "number",
+          "format": "float",
```

Comment:
Since the outside member(avsAssessment.properties.totalCpu) was already modelled as float. We have modelled the avsEstimatedNode.totalCPU to keep this consistent. If the parent class member starts to emit decimal values. the sub-class which is related to it should also start emitting the similar